pytools/__init__.py

```diff
@@ -1756,6 +1766,38 @@ def latex(self,
 
         return "\n".join(lines)
 
+    def raw(self) -> str:
```

Comment:
```suggestion
    def text_without_markup(self) -> str:
```
? (I'm not liking `raw` very much as a name.)